cmd/ltagent/init.go

```diff
@@ -64,6 +65,43 @@ func createChannels(admin *userentity.UserEntity, numChannels int) error {
 	return nil
 }
 
+func createTeamAdmins(admin *userentity.UserEntity, numUsers int, config *loadtest.Config) error {
+	for i := 0; i < numUsers; i++ {
+		index := i * 10 // making every 10th user to be a team admin
+		ueConfig := userentity.Config{
+			ServerURL:    config.ConnectionConfiguration.ServerURL,
+			WebSocketURL: config.ConnectionConfiguration.WebSocketURL,
+			Username:     fmt.Sprintf("testuser-%d", index),
+			Email:        fmt.Sprintf("testuser-%d@example.com", index),
+			Password:     "testPass123$",
+		}
+		store := memstore.New()
+		u := userentity.New(store, ueConfig)
+
+		err := u.SignUp(ueConfig.Email, ueConfig.Username, ueConfig.Password)
+		if err != nil {
+			mlog.Warn("error while signing up", mlog.Err(err)) // Possibly, user already exists.
+			continue
+		}
+		id := u.Store().Id()
+
+		ok, err := admin.IsSysAdmin()
+		if err != nil {
+			return err
+		}
+		if !ok {
+			return errors.New("admin user is not a sysadmin")
+		}
```

Comment:
Is this needed? We just created Teams and Channels, the admin is passed explicitly as admin.

Reply:
Guess not .. :)